hikari/impl/entity_factory.py

```diff
@@ -1759,6 +1759,8 @@ def _deserialize_command_option(self, payload: data_binding.JSONObject) -> comma
             autocomplete=payload.get("autocomplete", False),
             min_value=payload.get("min_value"),
             max_value=payload.get("max_value"),
+            name_localizations=payload.get("name_localizations", locales.Locale),
```

Comment:
This does not look correct? Were you perhaps trying to cast and managed to set a default value?

Reply:
No, I wasn't trying to cast it. I didn't know what to set as default value for this. Should it just be `undefined.UNDEFINED`?

Reply:
It should ideally be an empty mapping.